src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,140 @@
+import * as React from 'react';
+import classnames from 'classnames';
+import { getCommonProps } from '../../utils';
+import { px } from '../../utils';
+
+import Select from '../Select/Select';
+import Button from '../Button/Button';
+import chevronRight from '../../assets/chevronRight.svg';
+
+export interface PaginationProps {
+  /**
+   * Unique id for component testing
+   */
+  id: string;
+  /**
+   * Base class for Pagination component.
+   */
+  className?: string;
+  /**
+   * Child element pass in to display as options to select from.
+   */
+  children?: React.ReactNode;
+  /**
+   * Boolean to specify whether the `<Pagination>` should be disabled
+   */
+  isDisabled?: boolean;
+  /**
+   * Optional text to display when disable is true
+   */
+  disabledText?: string;
+  /**
+   * Optional `onClick` handler that is called whenever the `<Pagination>` value is clicked
+   */
+  onClick?: (event: React.MouseEvent<HTMLElement>) => void;
+  /**
+   * Optional `onChange` handler that is called whenever the `<Pagination>` value is changed
+   */
+  onChange?: (selectedValue: string) => void;
+  /**
+   * Specify the value of the `<Pagination>`
+   */
+  value?: string | undefined;
+  /**
+   * The default Inline style to apply to the `<Pagination>` Select component
+   */
+  variant?: 'inline';
+}
+
+/**
+ * ## Overview
+ *
+ * A component for adding a Pagination component.
+ *
+ * [Figma Link](https://www.figma.com/design/hxqgsE26wM7hII0WaUaDfF/RW---TIMED-Lot-Details-(PDP)?node-id=1-2&node-type=CANVAS&m=dev)
+ *
+ * [Storybook Link](https://phillips-seldon.netlify.app/?path=/docs/components-pagination--overview)
+ */
+
+const Pagination = ({
+  className,
+  onChange,
+  onClick,
+  variant = 'inline',
+  disabledText = 'Lot',
+  children,
+  ...props
+}: PaginationProps) => {
+  const type = 'pagination';
+  const { className: baseClassName, ...commonProps } = getCommonProps(props, 'Pagination');
+  const { id, value, isDisabled } = props;
+  const [selectedValue, setSelectedValue] = React.useState(value);
+  const selectRef = React.useRef<HTMLSelectElement>(null);
+
+  const paginationClick = (e: React.MouseEvent<HTMLElement>) => {
```

Comment:
could this be split out into `onPrevClick` and `onNextClick` functions?

might be a little more readable, and require less conditional branching

Reply:
I would prefer this: 

```suggestion
  const paginationClick = (newValue:string, e: React.MouseEvent<HTMLElement>) => {
```